README.md

```diff
@@ -85,13 +91,79 @@ reserver --version
 ```
 #### Reserve in test PyPI (test.pypi.org)
 ```console
-reserver --name sample_name1 sample_name2 --token=PYPI_TOKEN --test
+reserver --name sample_name1 sample_name2 --token=TEST_PYPI_TOKEN --test
 ```
 #### Reserve in main PyPI (pypi.org)
 ```console
 reserver --name sample_name1 sample_name2 --token=PYPI_TOKEN
 ```
+You can customize package parameters (listed below) by passing considered params as JSON file(s).
+
+<table>
+<tr>
+<th>Parameter</th>
+<th>Type</th>
+<th>Default</th>
+</tr>
+<td><code>description</code></td>
+<td>string</td>
+<td><code>This name has been reserved using Reserver</code></td>
+</tr>
+<td><code>author</code></td>
+<td>string</td>
+<td><code>Development Team</code></td>
+</tr>
+
+<td><code>author_email</code></td>
+<td>email address</td>
+<td><code>test@test.com</code></td>
+</tr>
+
+<td><code>url</code></td>
+<td>web address</td>
+<td><code>https://url.com</code></td>
+</tr>
+
+<td><code>download_url</code></td>
+<td>web address</td>
+<td><code>https://download_url.com</code></td>
+</tr>
+
+<td><code>source</code></td>
+<td>web address</td>
+<td><code>https://github.com/source</code></td>
+</tr>
+
+<td><code>license</code></td>
+<td>string</td>
+<td><code>MIT</code></td>
+</tr>
+
+</table>
 
+#### Custom parameters (one `param.json` for all packages)
```

Comment:
Long title!

My suggestion:

```
#### Custom parameters
...
Then we should pass `config.json` to the `--param` field of Reserver's CLI in order to get things done. 
Example1
Similarily, there should be per file, a dedicated JSON file containing associted parameters and then we need to pass them all to `--param` field of Reserver's CLI.
Example2
...

```

Reply:
OK, I update the way I've written this part, hope this version finally addresses your concerns.